CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.10.28-dev0
+
+### Enhancements
+
+* **Separated integration tests to reduce test time** Used `pytest` markers to mark long-running integration tests, to facilitate speedier local testing and to run separately in CI.
```

Comment:
i'm not sure "integration" is the right adjective here

Reply:
why not just call it "chipper," like it is?
also makes sense to move chipper tests to their own file IMO

Reply:
Not all the tests I've marked are chipper.